### DISCUSSION TITLE
Damaver summary results

### DISCUSSION THREAD

**Discussion by skmkhdjh** [biosaxs-com/atsas-community#1957] [Ab initio shape determination]
Hi, good morning. I need help with using Damaver to give a summary of the GASBOR models. 
I've shared the below pictures, and the problem arises. 

![Damaver control panel 1 ](https://github.com/user-attachments/assets/7b89bfee-5a50-4233-bc7a-7ca52c48573a)
![Damaver control panel 2](https://github.com/user-attachments/assets/18db5b0e-d4fd-4443-b358-2bbc385357d4)

Thank you so much for your attention and participation.

**Comment by franke-biosaxs** [comment]
Just for food for thought: gasbor writes Dummy Residue (DR) models. While *DAM*aver works on Dummy Atom Models. The difference being that the former does not have a grid, but the locations are variable, the latter works on a grid.

If you insist on using damaver on these, you have to first convert the Dummy Residue models to Dummy Atom Models via `cifop --dam=R dr.cif -o dam.cif` where R is the radius to use for the Dummy Atoms.

**Reply by skmkhdjh** [reply]
Thank you for your suggestion. I have tried to use the method above, however,
why does it say that it is unable to read the input file dam=R?

**Reply by franke-biosaxs** [reply]
Not "dam=R", but "--dam=R" and you have to replace the "R" with the radius of dummy atom beads you want to use.
